### PR TITLE
Add Python 3.8 as supported runtime

### PR DESCRIPTION
**Pull request by AutomatedTester**

### CODE REVIEW

**Comment by AutomatedTester** [comment]
Updated the pytest requirement as Travis was pulling an older version for some reason.

File: setup.py [hunk 1]
```diff
@@ -31,6 +31,7 @@
         'Programming Language :: Python :: 3.5',
```
Comment (belm0) [inline setup.py:31]: @mehaase there is no CI coverage for 3.5

Comment (AutomatedTester) [reply setup.py:31]: I did see there is #112 for that and I am happy to add it into this PR to get rid of it unless you want to support it. I only need 3.6+

Comment (belm0) [reply setup.py:31]: yes I forgot about #112 and PR #84

PR contribution is welcome, but that should be a separate PR

**Comment by belm0** [comment]
> Updated the pytest requirement as Travis was pulling an older version for some reason.

Well, the CI dependencies are not pinned.  Opened #137.

But yes, I recall that 3.8 needs a more recent pytest-- I assume you were hitting that.

**Comment by AutomatedTester** [comment]
> > Updated the pytest requirement as Travis was pulling an older version for some reason.
> 
> Well, the CI dependencies are not pinned. Opened #137.
> 
> But yes, I recall that 3.8 needs a more recent pytest-- I assume you were hitting that.

It's not a 3.8 issue per se but the latest `pytest-cov` need 4.6 or later. I didn't hit it locally since I got the latest version from pypi but weirdly travis seemed to pick up pytest 4.3 so it failed.

File: setup.py (hunk 1 above)
Comment (AutomatedTester) [reply setup.py:31]: Raised #138

**Pull request merged**